js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html

```diff
@@ -139,6 +168,52 @@ <h3 data-bind="css: $component.classes('analysis-name'), text: ($data.domainId ?
                             <!-- /ko -->
                         <!-- /ko -->
                     </div>
+                    <!-- ko if: reports.length !== 2 -->
```

Comment:
What is this for?

Reply:
Just to follow common behavior stratified results has the comparative mode in case when two reports in characterization